### PR TITLE
Fix flake8

### PR DESCRIPTION
**Pull request by Xiangs18**
Will fix flake8, mypy, bandit in 3 different PRs.

### CODE REVIEW

**Comment by Xiangs18** [comment]
Will fix flake8, mypy, bandit in 3 different PRs.

**Comment by Xiangs18** [comment]
https://github.com/jgi-kbase/IDMappingService/blob/7fde579ab8397a0b763f6624438735de378b2d77/src/jgikbase/idmapping/cli.py#L166

Could you please confirm which argument is unused?

File: src/jgikbase/idmapping/config.py [hunk 1]
```diff
@@ -165,43 +180,64 @@ def _get_cfg_from_env(self) -> Path:
...
-                    ('Parameter {} in configuration file {}, section {}, has whitespace-only entry'
-                     ).format(param_name, config[self._TEMP_KEY_CFG_FILE], self.CFG_SEC, str(e))
-                    ) from e
+                    (
+                        "Parameter {} in configuration file {}, section {}, has whitespace-only entry"
+                    ).format(
+                        param_name,
+                        config[self._TEMP_KEY_CFG_FILE],
+                        self.CFG_SEC,
+                        str(e),
```
Comment (Xiangs18) [inline src/jgikbase/idmapping/config.py:224]: str(e) here is unused arg as well?

**Comment by MrCreosote** [comment]
Yeah, has to be

File: src/jgikbase/idmapping/config.py (hunk 1 above)
Comment (MrCreosote) [reply src/jgikbase/idmapping/config.py:224]: Yep, seems that way. 

Flake8 must've changed a lot since the last time tests passed...

Comment (Xiangs18) [reply src/jgikbase/idmapping/config.py:224]: ah, I seriously doubt the last time tests all passed.

Comment (MrCreosote) [reply src/jgikbase/idmapping/config.py:224]: They did: https://github.com/jgi-kbase/IDMappingService/pull/126

File: src/jgikbase/idmapping/core/errors.py [hunk 2]
```diff
@@ -12,43 +13,46 @@ class ErrorType(Enum):
...
-    NO_TOKEN =               (10010, "No authentication token")  # noqa: E222 @IgnorePep8
+    NO_TOKEN = (10010, "No authentication token")  # noqa: E222 @IgnorePep8
     """ No token was provided when required. """
 
-    INVALID_TOKEN =          (10020, "Invalid token")  # noqa: E222 @IgnorePep8
+    INVALID_TOKEN = (10020, "Invalid token")  # noqa: E222 @IgnorePep8
     """ The token provided is not valid. """
 
-    UNAUTHORIZED =           (20000, "Unauthorized")  # noqa: E222 @IgnorePep8
+    UNAUTHORIZED = (20000, "Unauthorized")  # noqa: E222 @IgnorePep8
```
Comment (MrCreosote) [inline src/jgikbase/idmapping/core/errors.py:25]: This section more readable when the error codes are all whitespace aligned. I think the noqa should stop fake from complaining

File: src/jgikbase/idmapping/userlookup/kbase_user_lookup.py [hunk 3]
```diff
@@ -61,7 +61,7 @@ def _check_error(self, r):
         if r.status_code != 200:
             try:
                 j = r.json()
-            except Exception as e:  # @UnusedVariable
+            except Exception:  # @UnusedVariable
```
Comment (MrCreosote) [inline src/jgikbase/idmapping/userlookup/kbase_user_lookup.py:64]:
```suggestion
            except Exception:
```

**Comment by Xiangs18** [comment]
> Is this just running an automated formatter on the code? Are there any manual changes?

Automated formatter and manual changes are in 2 different commits.

File: src/jgikbase/idmapping/config.py (hunk 1 above)
Comment (Xiangs18) [reply src/jgikbase/idmapping/config.py]: 👍

File: src/jgikbase/idmapping/core/errors.py (hunk 2 above)
Comment (Xiangs18) [reply src/jgikbase/idmapping/core/errors.py]: 👍

File: src/jgikbase/idmapping/userlookup/kbase_user_lookup.py (hunk 3 above)
Comment (Xiangs18) [reply src/jgikbase/idmapping/userlookup/kbase_user_lookup.py]: 👍

**Review by MrCreosote — APPROVED** [review]

**Pull request merged**